tests/v2/validation/rbac/psa/psa.go

```diff
@@ -1,18 +1,15 @@
-package rbac
+package psa
```

Comment:
this is a little confusing to me. Why is etcdbackup not its own suite, but psa is?

Reply:
thanks for catching that. it was a missed step. Updated the package now.